docs/guides/developer/docs/develop/debugging.md

```diff
@@ -1,12 +1,59 @@
-# In Construction
+# Debugging in Opencast
 
-Attaching a Remote Debugger to Karaf
-------------------------------------
+This guide will show you how to debug Opencast.
 
-To debug a running Opencast system, you can attach a remote debugger in your IDE (Eclipse or NetBeans, i.e.). For that
+
+## Debugging strategies
+
+There are different ways to debug Opencast. You can debug the backend and the frontend.
+The backend is the Java code while the frontend is the Angular code.
```

Comment:
Angular is departing the codebase, hopefully soon.  The new one is TypeScript, IIRC.